packages/ai-mcp/src/node/mcp-server.ts

```diff
@@ -39,22 +55,40 @@ export class MCPServer {
         if (this.started) {
             return;
         }
-        console.log(`Starting server "${this.name}" with command: ${this.command} and args: ${this.args?.join(' ')} and env: ${JSON.stringify(this.env)}`);
-        // Filter process.env to exclude undefined values
-        const sanitizedEnv: Record<string, string> = Object.fromEntries(
-            Object.entries(process.env).filter((entry): entry is [string, string] => entry[1] !== undefined)
-        );
-
-        const mergedEnv: Record<string, string> = {
-            ...sanitizedEnv,
-            ...(this.env || {})
-        };
-        const transport = new StdioClientTransport({
-            command: this.command,
-            args: this.args,
-            env: mergedEnv,
-        });
-        transport.onerror = error => {
+
+        console.log(`Starting server "${this.name}" with transport: ${this.transportType}`);
```

Comment:
We have two messages now instead of one. I would prefer having only one, i.e. augment the specific ones below with a common string from here.